### PR TITLE
Added commands for asking what song is playing

### PR DESCRIPTION
**Pull request by moritzkraemer**
#15

### CODE REVIEW

File: src/DankDitties/VoiceAssistantWorker.cs [hunk 1]
```diff
@@ -133,7 +133,7 @@ protected override async Task DoWorkAsync(CancellationToken cancellationToken)
                                         }
                                     }
                                 }
-                                else if (text.ToLower() == "what song is this")
+                                else if (text.ToLower() == "what song is this" || text.ToLower() == "what's playing" || text.ToLower() == "song" || text.ToLower() == "song name" || text.ToLower() == "damn son whered you find this")
```
Comment (prmichaelsen) [inline src/DankDitties/VoiceAssistantWorker.cs:136]: > `damn son whered you find this`

**Comment by monodop** [comment]
Nice, how did you find out about this repo?

**Comment by WilliamRoyNelson** [comment]
Damn son, how'd you find this?

**Review by monodop — APPROVED** [review]

**Comment by moritzkraemer** [comment]
Was just scrolling through issues and found this and thought i could do that :D (for hacktoberfest so i would love if you could add the "hacktoberfest-accepted" tag to this PR)

**Comment by monodop** [comment]
I added the tag to the repo. let me know if it counted

**Comment by moritzkraemer** [comment]
Thanks alot it counted!

**Pull request merged**